mobile/src/bundles/common/styles/colors.ts

```diff
@@ -0,0 +1,13 @@
+enum Color {
+    PRIMARY = '#274F8D',
+    SECONDARY = '#18A0FB',
+    ERROR = '#FF2121',
+    HOVER = '#0084DD',
+    BACKGROUND = '#F7F8FC',
+    INPUT = '#D5DCE8',
+    TEXT2 = '#90A3BF',
+    TEXTDARK = '#7C8798',
+    TAG = '#DCE5FF',
+}
```

Comment:
We discussed switching from ts enums to js enums with coaches, so let's replace with js enum 
```suggestion
const Color = {
    PRIMARY: '#274F8D',
    SECONDARY: '#18A0FB',
    ...
} as const;
```